### DISCUSSION TITLE
Save show-completed values for each list

### DISCUSSION THREAD

**Discussion by danirabbit** [elementary/tasks#221] [Ideas]
Does this make sense? Is there a case where we want one list to show only incomplete items but another list should show all items?

**Comment by marbetschar** [comment]
I always was happy with the possibility to show either all items or only the incompleted ones within the same list by setting a list filter. Don't see added value in dedicated lists or the possibility to show only completed items. I'm open to suggestions though.

**UPDATE:** I think I misunderstood the question. Saving the "show-completed" filter per list was actually what I expected the app to do. But right now it doesn't. To be fair, I have to say that I'm very used to the Apple Reminders app, and there it is a per list setting. Don't have very strong feelings about it though, I think both perspectives do have their use cases.
